socorro/cron/jobs/verify_processed.py

```diff
@@ -9,55 +9,41 @@
 crash files.
 """
 
+import concurrent.futures
 import datetime
 from functools import partial
-import logging
-import multiprocessing
-import sys
 
 from configman import Namespace, class_converter
 import markus
-import more_itertools
 from psycopg2 import IntegrityError
 
 from socorro.cron.base import BaseCronApp
 from socorro.cron.mixins import as_backfill_cron_app
 from socorro.lib.dbutil import execute_no_results
-from socorro.lib.raven_client import capture_error
 from socorro.lib.transaction import transaction_context
 
 
 metrics = markus.get_metrics('crontabber.verifyprocessed')
 
 
-def check_crashids(entropy, boto_conn, bucket_name, date, sentry_dsn):
+def check_crashids(entropy, boto_conn, bucket_name, date):
```

Comment:
With `concurrent.futures`, child processes will send exceptions back to the parent to handle. So we don't need the extra error handling here.